app/src/main/kotlin/com/thundermaps/apilib/android/api/resources/TeamUsersResource.kt

```diff
@@ -0,0 +1,12 @@
+package com.thundermaps.apilib.android.api.resources
+
+import com.thundermaps.apilib.android.api.requests.RequestParameters
+import com.thundermaps.apilib.android.api.responses.models.Result
+import com.thundermaps.apilib.android.api.responses.models.TeamUserDetails
+
+interface TeamUsersResource {
```

Comment:
Team Resource can provide the TeamUser. We can update TeamResource to get the list of TeamUser

Reply:
Ok will add it into the TeamResource